includes/types/CMB2_Type_File_List.php

```diff
@@ -55,7 +87,7 @@ public function render() {
 				if ( $this->is_valid_img_ext( $fullurl ) ) {
 
 					$output .= $this->img_status_output( array(
-						'image'    => wp_get_attachment_image( $id, $img_size ),
+						'image'    => wp_get_attachment_image( $id, $img_size, null, array( 'class' => 'cmb-file_list-field-image', 'style' => 'height: auto;' ) ),
```

Comment:
The 'auto' height is needed to stop the images being squashed if they're over the max-height that's set in the css, adding it here is probably just lazy though. I'll look at just adding it to the existing css.